pages/visits/end.js

```diff
@@ -50,10 +50,22 @@ const End = ({ wardId, callId, surveyUrl }) => {
               </h1>
 
               <div className="nhsuk-panel__body">
-                <p>Thank you for using the virtual visit service.</p>
-                <p>Your personal data will be removed within 24 hours.</p>
+                <p className="nhsuk-u-margin-bottom-0">
+                  Thank you for using the virtual visit service.
+                </p>
               </div>
             </div>
+
+            <h2>What happens next</h2>
+
+            <p>Your personal data will be removed within 24 hours.</p>
+
+            {supportUrl && (
+              <ActionLink href={supportUrl}>
+                Get support from this hospital
+              </ActionLink>
+            )}
+
```

Comment:
Do you think it's worth wrapping this, and the similar code below, up in a component?

Reply:
Are you thinking like a `WardVisitEnd` component for the ward staff and then `VisitorVisitEnd` component for the visitor/key contact?

Reply:
Sorry, I failed to select the full section of code for the comment, which would've made it clearer.

I'm talking that having an `ExtraInformation` component, or something, would be worthwhile as the _What happens Next_ and _Help improve virtual visits_ sections are the same. Having them in a component would mean that they stay consistent and all you'd need to do is pass in the text to populate it.

Reply:
Ah okay! Something like the below?

```jsx
<ActionLinkSection heading="What happens next" link={supportUrl} />

<ActionLinkSection heading="Help improve virtual visits" link={surveyUrl}>
  <p>
    We’d welcome your feedback. Can you answer some questions
    about your virtual visit today?
  </p>
</ActionLinkSection>
```

Reply:
Pushed up two new commits with this, let me know what you think!

Reply:
Late to the party, but that's perfect!